etl/steps/data/garden/wb/2024-01-17/world_bank_pip.py

```diff
@@ -0,0 +1,1047 @@
+        "poverty_gap_index_50_median",
+        "poverty_gap_index_60_median",
+    ]
+    tb.loc[:, pct_indicators] = tb[pct_indicators] * 100
+
+    # Create a new column for the poverty line in cents and string
+    tb["poverty_line_cents"] = round(tb["poverty_line"] * 100).astype(int).astype(str)
+
+    # Make the table wide, with poverty_line_cents as columns
+    tb = tb.pivot(
+        index=[
+            "ppp_version",
+            "country",
+            "year",
+            "reporting_level",
+            "welfare_type",
+            "survey_comparability",
+            "comparable_spell",
+            "reporting_pop",
+            "mean",
+            "median",
+            "mld",
+            "gini",
+            "polarization",
+            "decile1_share",
+            "decile2_share",
+            "decile3_share",
+            "decile4_share",
+            "decile5_share",
+            "decile6_share",
+            "decile7_share",
+            "decile8_share",
+            "decile9_share",
+            "decile10_share",
+            "decile1_thr",
+            "decile2_thr",
+            "decile3_thr",
+            "decile4_thr",
+            "decile5_thr",
+            "decile6_thr",
+            "decile7_thr",
+            "decile8_thr",
+            "decile9_thr",
+            "is_interpolated",
+            "distribution_type",
+            "estimation_type",
+            "headcount_40_median",
+            "headcount_50_median",
+            "headcount_60_median",
+            "headcount_ratio_40_median",
+            "headcount_ratio_50_median",
+            "headcount_ratio_60_median",
+            "income_gap_ratio_40_median",
+            "income_gap_ratio_50_median",
+            "income_gap_ratio_60_median",
+            "poverty_gap_index_40_median",
+            "poverty_gap_index_50_median",
+            "poverty_gap_index_60_median",
+            "avg_shortfall_40_median",
+            "avg_shortfall_50_median",
+            "avg_shortfall_60_median",
+            "total_shortfall_40_median",
+            "total_shortfall_50_median",
+            "total_shortfall_60_median",
+            "poverty_severity_40_median",
+            "poverty_severity_50_median",
+            "poverty_severity_60_median",
+            "watts_40_median",
+            "watts_50_median",
+            "watts_60_median",
+        ],
+        columns="poverty_line_cents",
+        values=[
+            "headcount",
+            "headcount_ratio",
+            "income_gap_ratio",
+            "poverty_gap_index",
+            "avg_shortfall",
+            "total_shortfall",
+            "poverty_severity",
+            "watts",
+        ],
+    )
+
+    # Flatten column names
+    tb.columns = ["_".join(col).strip() for col in tb.columns.values]
+
+    # Reset index
+    tb = tb.reset_index()
+
+    return tb
+
+
+def create_stacked_variables(tb: Table, povlines_dict: dict, ppp_version: int) -> tuple([Table, list, list]):
+    """
```

Comment:
The output type should be `Tuple[Table, list, list]` (use `from typing import Tuple`).